test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -290,6 +290,48 @@ class cursor_bound_01 : public test {
         return std::make_pair(lower_bound, upper_bound);
     }
 
+    /*
+     * Validate the bound search call. If the key is within range, the range cursor should return
+     * back the search key.
+     */
+    void
+    validate_bound_search(int range_ret, scoped_cursor &range_cursor, scoped_cursor &normal_cursor,
```

Comment:
I think we don't need to pass in the normal cursor anymore 🍔

Reply:
You're right, thanks!